### PR TITLE
Update Condition with new params and response examples

### PR DESCRIPTION
**Pull request by daliboz**
Updated Condition with new params and response examples, updated to indicate which fields are required.

### CODE REVIEW

File: lib/resources.rb [hunk 1]
```diff
@@ -1132,37 +1132,194 @@ def get_resource(key)
...
-                                                           "code" => "424300006",
-                                                           "display" => "Minimal shared activities"}],
-                                             "text" => "Minimal shared activities"},
-                                  "clinicalStatus" => "confirmed",
-                                  "onsetDateTime" => "2007"}}]
-    }
+        "resourceType"=> "Bundle",
+        "id"=> "d457d75f-4d8d-439e-9aca-2d8294457652",
+        "type"=> "searchset",
+        "base"=> "https=>//fhir.sandboxcernerpowerchart.com/may2015/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
```
Comment (jfrhong) [inline lib/resources.rb]: 'https=>" to "https:"

File: lib/resources.rb [hunk 2]
```diff
@@ -1132,37 +1132,194 @@ def get_resource(key)
...
+        "total" => 86,
+        "link" => [{
+                       "relation"=> "self",
+                       "url"=> "https=>//fhir.sandboxcernerpowerchart.com/may2015/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Condition?patient=1316024"
```
Comment (jfrhong) [inline lib/resources.rb]: 'https=>" to "https:"

File: lib/resources.rb [hunk 3]
```diff
@@ -1132,37 +1132,194 @@ def get_resource(key)
...
-                                                           "display" => "Minimal shared activities"}],
-                                             "text" => "Minimal shared activities"},
-                                  "clinicalStatus" => "confirmed",
-                                  "onsetDateTime" => "2007"}}]
-    }
+        "resourceType"=> "Bundle",
+        "id"=> "d457d75f-4d8d-439e-9aca-2d8294457652",
+        "type"=> "searchset",
+        "base"=> "https=>//fhir.sandboxcernerpowerchart.com/may2015/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+        "total" => 86,
```
Comment (MaxPhilips) [inline lib/resources.rb]: Change to 5

File: lib/resources.rb [hunk 4]
```diff
@@ -1132,37 +1132,194 @@ def get_resource(key)
...
+                                "reference"=> "Encounter/1309918"
+                            },
+                            "asserter" => {
+                                "reference"=> "Practitioner/1418008",
+                                "display"=> "Brewer, Phil V"
+                            },
+                            "dateAsserted"=> "2013-02-08",
+                            "code" => {
+                                "coding" => [{
+                                                 "system"=> "http=>//hl7.org/fhir/sid/icd-9",
```
Comment (MaxPhilips) [inline lib/resources.rb]: => to : here, 1179

File: lib/resources.rb [hunk 5]
```diff
@@ -1132,37 +1132,194 @@ def get_resource(key)
...
+                        "resource" => {
+                            "resourceType"=> "Condition",
+                            "id"=> "d21591516",
+                            "meta" => {
+                                "versionId"=> "21591516",
+                                "lastUpdated"=> "2013-02-08T18:03:09.000Z"
+                            },
+                            "text" => {
+                                "status"=> "generated",
+                                "div"=> "&lt;div>&lt;p>&lt;b>Condition&lt;/b>&lt;/p>&lt;p>&lt;b>Patient&lt;/b>: PETERS, TIMOTHY&lt;/p>&lt;p>&lt;b>Category&lt;/b>: Diagnosis&lt;/p>&lt;p>&lt;b>Code&lt;/b>: Absence seizures, 345.0&lt;/p>&lt;p>&lt;b>Clinical Status&lt;/b>: Confirmed&lt;/p>&lt;p>&lt;b>Severity&lt;/b>: &lt;/p>&lt;p>&lt;b>Onset&lt;/b>: 2009-05-04&lt;/p>&lt;/div>"
```
Comment (MaxPhilips) [inline lib/resources.rb]: Same comment as AllergyIntolerance PR about writing these consistently although they display fine

**Comment by MaxPhilips** [comment]
Required does not have a question mark here but does in the AllergyIntolerance PR.

**Comment by MaxPhilips** [comment]
When the Required entries have more than just a Y or N I don't think the non-monospace looks as bad.

**Comment by MaxPhilips** [comment]
+1 after resources.rb updates and Required column title change (either here or in https://github.com/cerner/fhir.cerner.com/pull/18)

**Comment by daliboz** [comment]
updated per comments

**Comment by jfrhong** [comment]
+1

**Comment by dennispatterson** [comment]
Similar to Allergies, replace associate names?

**Comment by daliboz** [comment]
updated names

**Comment by dennispatterson** [comment]
:+1:

**Pull request merged**